package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "stylelint-config-visionapps-order",
-  "version": "0.3.1",
+  "name": "@visionappscz/stylelint-config-visionapps-order",
```

Comment:
@bedrich-schindler You did the same thing (adding namespace) with [eslint-config-visionapps](https://github.com/visionappscz/eslint-config-visionapps). Is there anything else needed?

Reply:
@adamkudrna This is the only thing that need to be changed in code. So it is correct. But when new version is released, please deprecate the old one like I did it with eslint-visionapps-config.

Docs: https://docs.npmjs.com/cli/deprecate